api.rest

```diff
@@ -65,14 +65,30 @@ x-bp-messaging-client-token: {{clientToken}}
 Content-Type: application/json
 
 {
-  "collect": false,
   "conversationId": "{{convoId}}",
+  "authorId": null,
   "payload": {
     "type": "text",
     "text": "Hello this is a text message!"
   }
 }
 
+### Converse
+POST {{baseUrl}}/api/messages/collect
+Authorization: Bearer {{authToken}}
+x-bp-messaging-client-id: {{clientId}}
+x-bp-messaging-client-token: {{clientToken}}
+Content-Type: application/json
+
+{
+  "conversationId": "{{convoId}}",
+  "authorId": "{{userId}}",
+  "payload": {
+    "type": "text",
+    "text": "Hello this is a converse message!"
+  }
+}
+
```

Comment:
This is the new route for converse. It's not a parameter in the query anymore since the behavior was way too different.